blog/config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: sqlite3
+  pool: 5
+  timeout: 5000
```

Comment:
Echoing my earlier note about postgresql, you would need to update this file manually if you didn't set your application up with postgres.